ophyd/areadetector/filestore_mixins.py

```diff
@@ -74,7 +83,6 @@ def stage(self):
         path = os.path.join(*(date.isoformat().split('-') + ['']))
         full_write_path = os.path.join(self.write_file_path, path)
         full_read_path = os.path.join(self.read_file_path, path)
```

Comment:
Would it be possible to break out this path building into a method that can be overridden?

Because you're going to ask: in the case of tiff files where we save 300x300 = 90,000 files per scan at the HXN, we will get extreme delays when attempting to list these files over NFS. Separate directories per-scan (i.e., per-stage) would help. (And before you say 'HDF5!', it's just not feasible to move to it currently for other reasons)